assets/stylesheets/modules/_promotion.scss

```diff
@@ -0,0 +1,97 @@
+.promotion__landing-page {
```

Comment:
According to BEM convention, this is a child selector which describes the `landing-page` child of a `promotion` component. But where is the promotion component? Similarly for all other rules here: we have lots of children but no parent.